config/mongoid.yml

```diff
@@ -81,3 +81,13 @@ test:
         # low amounts for fast failures.
         max_retries: 1
         retry_interval: 0
+
+production:
+  sessions:
+    default:
+      uri: <%= ENV['MONGODB_URI'] %>
+      options:
+        logger: false
+        use_activesupport_time_zone: true
+        refresh_interval: 120
+        refresh_mode: :sync
```

Comment:
I've added issue #958 to prompt us to investigate removing this configuration option. I think I prefer that to adding a comment to the config.